### PR TITLE
Fix DG Diagrams

### LINKED ISSUE #307: Fix Developer Guide diagrams
The following diagrams have mistakes and their .puml files are not in the repo

1. Find doctor/patient activity diagram YZ inconsistent label
image

2.  List doctor/patient sequence diagram
image

3. Delete tag sequence diagram
image

### PR DESCRIPTION
**Pull request by kimshitong**
Closes #307

### CODE REVIEW

**Review by derekjxtan — APPROVED** [review]

File: docs/images/DeleteSpecialisationSequenceDiagram.png
[the remark this answers is not in the document]
Comment (derekjxtan) [reply docs/images/DeleteSpecialisationSequenceDiagram.png:1]: the constructor takes in type Tag not String tho

[the remark this answers is not in the document]
Comment (derekjxtan) [reply docs/images/DeleteSpecialisationSequenceDiagram.png:1]: @kimshitong change to `DeleteSpecialisationCommand()` instead cos it takes in types Index and Tag as arguments. Help to reflect this changes in the `Add specialisation/medical condition/prescription` part also

[the remark this answers is not in the document]
Comment (kimshitong) [reply docs/images/DeleteSpecialisationSequenceDiagram.png:1]: Ok, thanks.

just noticed

image
Edit Remark, do i change this oso?

[the remark this answers is not in the document]
Comment (mounilsankar) [reply docs/images/DeleteSpecialisationSequenceDiagram.png:1]: should change

**Review by derekjxtan — APPROVED** [review]

**Pull request merged**